src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessSink.cs

```diff
@@ -4,40 +4,72 @@
-    public class ExceptionLessSink : ILogEventSink {
+    public class ExceptionlessSink : ILogEventSink {
         private readonly Func<EventBuilder, EventBuilder> _additionalOperation;
         private readonly bool _includeProperties;
 
-        private ExceptionlessClient _client = ExceptionlessClient.Default;
+        private readonly ExceptionlessClient _client;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="ExceptionLessSink"/> class.
+        /// Initializes a new instance of the <see cref="ExceptionlessSink"/> class.
         /// </summary>
+        /// <param name="apiKey">
+        /// The API key that will be used when sending events to the server.
+        /// </param>
+        /// <param name="serverUrl">
+        /// Optional URL of the server events will be sent to.
+        /// </param>
         /// <param name="additionalOperation">
-        /// Optional operation to run against the Error Builder before submitting to ExceptionLess
+        /// Optional operation to run against the Error Builder before submitting to Exceptionless
         /// </param>
         /// <param name="includeProperties">
-        /// If false then the seri log properties will not be submitted to ExceptionLess
+        /// If false then the Serilog properties will not be submitted to Exceptionless
         /// </param>
-        public ExceptionLessSink(Func<EventBuilder, EventBuilder> additionalOperation = null, bool includeProperties = true) {
+        public ExceptionlessSink(
+            string apiKey,
+            string serverUrl = null,
+            Func<EventBuilder, EventBuilder> additionalOperation = null,
+            bool includeProperties = true
+        ) {
+            if (apiKey == null)
+                throw new ArgumentNullException(nameof(apiKey));
+
+            _client = new ExceptionlessClient(cfg => {
+                if (!String.IsNullOrEmpty(apiKey)) {
+                    cfg.ApiKey = apiKey;
+                }
+                if (!String.IsNullOrEmpty(serverUrl)) {
+                    cfg.ServerUrl = serverUrl;
+                }
+                cfg.UseInMemoryStorage();
+            });            
             _additionalOperation = additionalOperation;
             _includeProperties = includeProperties;
-        
-            // TODO: Read this from configuration.
-            //if (String.IsNullOrEmpty(ApiKey) && String.IsNullOrEmpty(ServerUrl))
-            //    return;
+        }
 
-            //_client = new ExceptionlessClient(config => {
-            //    if (!String.IsNullOrEmpty(ApiKey))
-            //        config.ApiKey = ApiKey;
-            //    if (!String.IsNullOrEmpty(ServerUrl))
-            //        config.ServerUrl = ServerUrl;
-            //    config.UseInMemoryStorage();
-            //});
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExceptionlessSink"/> class.
+        /// </summary>
+        /// <param name="client">
```

Comment:
I'm thinking we might want to move this to the last parameter as it makes it a bit easier to use if you are using the default client.